custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -327,6 +338,93 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+class NikshayRegisterPrivatePatientPayloadGenerator(BaseNikshayPayloadGenerator):
+    format_name = 'case_xml'
+    format_label = 'XML'
+
+    @memoized
```

Comment:
Thank you @proteusvacuum . Was just thinking of making something more generic which can be used in case payload is needed by any other repeater. That change for just `_get_person_locations` does do the needful.